### PR TITLE
Add support for smaller terminal QR codes

### PR DESCRIPTION
**Pull request by bambooCZ**
Renderer for smaller QR codes into terminal.

### CODE REVIEW

**Comment by bambooCZ** [comment]
Snímek z 2020-05-26 17-18-16

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Renderer options:
   -q, --qzone       Quiet zone size                                     [number]
   -l, --lightcolor  Light RGBA hex color
   -d, --darkcolor   Dark RGBA hex color
+  --small  Output smaller QR code to terminal                          [boolean]
```
Comment (pepakriz) [inline README.md:64]: Why not reuse `--scale`?

Comment (bambooCZ) [reply README.md:64]: No problem. What value would you suggest? Anything below 1 should use the smaller version?

**Comment by xiao-ben** [comment]
why not publish new version ? I want to use it .

**Comment by bambooCZ** [comment]
This repo is apparently dead, isn't it?

**Comment by FrankFan** [comment]
Great! 
I'm looking forward this awesome feature,
 please release to npm

**Comment by bambooCZ** [comment]
@L-Chris It uses UTF box-drawing characters to create the QR code string.

**Comment by soldair** [comment]
repo isnt dead. i just wandered off for a while

thanks for the contribution and ill try it out.

**Comment by timqian** [comment]
@soldair Looks this feature is not released on npm?

**Comment by soldair** [comment]
Not yet

**Pull request merged**